pages/linux/xcursorgen.md

```diff
@@ -0,0 +1,21 @@
+# xcursorgen
+
+> Create an X cursor file from a collection of PNG images.
```

Comment:
I would write it like that instead:

`Create cursor themes from images.`

Reply:
Why 'themes'?

Reply:
To me, the current description is very accurate.  Is it not?

Reply:
> Why 'themes'?

Because it creates cursor themes.

Reply:
Your proposed description sounds a bit short comparde to the original, though.  Any objections to this?
```suggestion
> Create an X cursor theme from a collection of PNG images.
```

Reply:
> Because it creates cursor themes.

Actually, one run of this command only creates one cursor and to create a theme, you need an `index.theme` file:
```
[Icon Theme]
Name = <name>
Comment = <comment>
Example = default
Inherits = <the old cursor theme>
```

So, I'd say this command does creates cursors, not themes